bitnami/grafana-tempo/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: grafana-tempo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/grafana-tempo
-version: 2.11.0
+version: 2.11.1
```

Although debatable, I would say this refactor is not really a fix and going by semver standards we should bump de the minor version of the chart. We also just merged https://github.com/bitnami/charts/pull/24091 so we have to take that into account as well

```suggestion
version: 2.13.0
```